.circleci/config.yml

```diff
@@ -1244,3 +1230,7 @@ workflows:
       - t_ubu_asan_constantinople: *workflow_ubuntu2004_asan
       - t_ubu_asan_constantinople_clang: *workflow_ubuntu2004_asan_clang
       - t_ubu_asan_cli: *workflow_ubuntu2004_asan
+
+      # Emscripten build and tests that take more than 15 minutes to execute
+      - b_ems: *workflow_trigger_on_tags
```

Comment:
Can we move b_ems to nightly? Isn't it a requirement for tons of tests? Also, I think it is really useful to have emscripten binaries for all pull requests.

Reply:
I didn't move it, just copied it - exactly because it's a requirement for tons of tests, including the external ones :)

Reply:
Ah ok!